packages/atomic/cypress/fixtures/test-fixture.ts

```diff
@@ -1,9 +1,22 @@
+import {Result} from '@coveo/headless';
 import {CyHttpMessages} from 'cypress/types/net-stubbing';
 import {i18n} from 'i18next';
 import {SearchResponseSuccess} from '../../../headless/dist/definitions/api/search/search/search-response';
 import {AnalyticsTracker, AnyEventRequest} from '../utils/analyticsUtils';
 import {buildTestUrl} from '../utils/setupComponent';
 
+export interface ResultWithFolding extends Result {
```

Comment:
I would not add any interface that relates to results inside `test-fixture.ts`

Reply:
this actually comes from headless, but doesn't seem to be exported there. would it make sense to change it in headless? I don't see anywhere else that it would fit in the current structure, would need to add a separate file maybe?

Reply:
I wouldn't export it from Headless, I think it could fit in `folded-result-list-actions.ts` (alongside the methods that need it).